gtda/externals/python/ripser_interface.py

```diff
@@ -21,10 +21,7 @@ def DRFDM(DParam, maxHomDim, thresh=-1, coeff=2, do_cocycles=0):
     else:
         ret = gtda_ripser_coeff.rips_dm(DParam, DParam.shape[0], coeff,
                                         maxHomDim, thresh, do_cocycles)
-    ret_rips = {}
-    ret_rips.update({"births_and_deaths_by_dim": ret.births_and_deaths_by_dim})
-    ret_rips.update({"num_edges": ret.num_edges})
-    return ret_rips
```

Comment:
It looks like you removed these because they were redundant information already contained in `ret`. Correct?

Reply:
Exactly, at the beginning when importing this library, I wanted to keep the same "behavior" as them. But in our case, it doesn't make sense because we have full control on the python ripser interface.